packages/augur-ui/src/modules/notifications/selectors/notification-state.ts

```diff
@@ -287,31 +225,41 @@ export const selectNotifications = createSelector(
       });
     }
 
-    if (proceedsToClaim && proceedsToClaim.length > 0) {
-      let totalEth = createBigNumber(0);
-
-      const marketIds = proceedsToClaim.map(market => market.id);
-      proceedsToClaim.forEach(market => {
-        totalEth = totalEth.plus(
-          createBigNumber(Number(market.outstandingReturns || 0))
+    const { accountPositions } = store.getState();
+    let totalDai = createBigNumber(0);
+    const marketIds = [];
+    for (const marketId in accountPositions) {
```

Comment:
we don't use many for loops, use `.reduce` to collect marketIds in an array and sum totalDai.

market.reduce((p, m) => {  .... }, {marketIds: [], totalDai: ZERO});